src/schema/v2/homeView/sections/QuickLinks.ts

```diff
@@ -146,3 +151,26 @@ export const QUICK_LINKS: Array<NavigationPill> = [
     icon: "FairIcon",
   },
 ]
+
+async function maybeInsertYourBidsLink(
+  links: NavigationPill[],
+  context: ResolverContext
+) {
+  if (isFeatureFlagEnabled("onyx_enable-quick-links-v2")) {
+    const { MyBids } = require("schema/v2/me/myBids")
+    const bids = await MyBids.resolve?.({}, {}, context, {} as any)
+    const hasBids = bids?.active?.length > 0
+    const auctionsIndex = links.findIndex((link) => link.href === "/auctions")
```

Comment:
Noting that once this is enabled, this `MyBids.resolve(…)` check will be performed on every home view request. We should keep an eye on performance.